convertCSV.js

```diff
@@ -0,0 +1,50 @@
+class CsvPhraseImporter {
+  constructor(event) {
+    this.readFile(event)
+  }
+  
+  readFile(event) {
+    let file = event.target.files[0]
+    
+    const reader = new FileReader();
+    reader.addEventListener('load', (event) => {
+      //console.log(csv)
+      this.phrase = this.csvToPhrase(event.target.result)
+    });
+    reader.readAsText(file);
+  }
+  
+  toPhrase = function(value, state) {
```

Comment:
This does the same thing
```suggestion
  toPhrase(value, state) {
```

Reply:
switched to a different parser so this callback function was no longer needed and has since been deleted.